tests/fixtures/ModelsPage.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, Locator, Page } from '@playwright/test';
+import { animals, colors, uniqueNamesGenerator } from 'unique-names-generator';
+
+export class ModelsPage {
+  readonly page: Page;
+
+  readonly jarPath: string;
+  readonly modelName: string;
+  readonly modelVersion: string;
+
+  readonly alertError: Locator;
+  readonly confirmModal: Locator;
+  readonly confirmModalDeleteButton: Locator;
+  readonly createButton: Locator;
+  readonly deleteButton: Locator;
+  readonly inputFile: Locator;
+  readonly inputName: Locator;
+  readonly inputVersion: Locator;
+  readonly modelTableRow: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    this.jarPath = 'tests/data/banananation-develop.jar';
```

Comment:
I assume that because these are unit tests that this jar is just a file to upload and so there is no worries about whether it's up to date with respect to to the latest mission model interface etc.?

Reply:
For right now I am the mechanism :)
I will add a more automated way to do this in a future commit after I finish transferring the remainder of the tests. Since we build the JAR in CI we can just set a path and use it directly. Will add a TODO in the meantime.

Reply:
haha makes sense! I wanted to make sure I was understanding what I was reading.